up-l1/README.adoc

```diff
@@ -476,17 +438,17 @@ API Signature:
 
 === Receive()
```

Comment:
How should it be implemented or used?
Is the receive waiting for the next message? for how long? 
Is it the last received message? who will maintain the last message? a listener? 
I would suggest that the receive will be the blocking solution that will have a timeout.

Reply:
Why? Can you please elaborate?

Reply:
I don't understand the when would you want to use this method. is it for getting the next message? is it to pick a message from a queue? or picking the last message (local uTwin)?
That is why I am looking at the performance issues of the options. The "local uTwin" or the "queue" option will need to queue messages using the callback function that would need to run in the background. how many messages do we queue? for how long?